### PR TITLE
408 web internationalization translation part

### PR DESCRIPTION
**Pull request by invpuppet**
Translate web application
image

No changes in tests other than fixes to support localization

### CODE REVIEW

File: Configurateur/front/lib/l10n/app_fr.arb [hunk 1]
```diff
@@ -1,3 +1,323 @@
...
+	"email": "Email",
+	"emailAddress": "Adresse e-mail",
+	"emailData": "Email : {email}",
+	"emailEdit": "Modification de votre adresse mail",
+	"emailEdited": "Email modifié avec succès !",
+	"emailFill": "Entrez votre email",
+	"emailNew": "Nouvel email",
+	"emailSendRecovery": "Envoyer l'email de récupération",
+	"emailUpdate": "Mettez à jour votre adresse mail facilement !",
+	"endConversation": "Cloturer la conversation",
```
Comment (hugolopespinto) [inline Configurateur/front/lib/l10n/app_fr.arb:116]: Clôturer

File: Configurateur/front/lib/l10n/app_fr.arb [hunk 2]
```diff
@@ -1,3 +1,323 @@
...
+	"objectPrice": "Prix de l'objet",
+	"opinion": "Avis",
+	"opinionConsultCreate": "Consulter et donner votre avis sur RISU !",
+	"opinionPage": "Page des avis",
+	"opinionPost": "Poster un avis",
+	"opinionPostedBy": "Avis posté par {firstName} {lastName}",
+	"opinionShare": "Partagez votre expérience en postant un avis !",
+	"opinionYours": "Vos avis",
+	"orderConfirmation": "Confirmation de commande",
+	"orderConfirmationReturnHome": "Votre commande a bien été confirmée, vous pouvez maintenant retournez à l'accueil et nous vous contacterons dès que votre commande sera prête",
```
Comment (hugolopespinto) [inline Configurateur/front/lib/l10n/app_fr.arb:221]: retourner

File: Configurateur/front/lib/l10n/app_fr.arb [hunk 3]
```diff
@@ -1,3 +1,323 @@
...
+	"registerResendEmail": "Renvoyer le mail de confirmation",
+	"registerRisu": "Inscrivez-vous sur le site RISU !",
+	"registerWith": "S'inscrire avec :",
+	"registration": "Inscription",
+	"remove": "Retirer",
+	"removeLocker": "Retirer un casier",
+	"returnTo": "Retour",
+	"right": "Droite",
+	"rightView": "vue droite",
+	"risuCompany": "L'entreprise Risu",
```
Comment (hugolopespinto) [inline Configurateur/front/lib/l10n/app_fr.arb:275]: Risu or RISU ?

File: Configurateur/front/lib/l10n/app_fr.arb [hunk 4]
```diff
@@ -1,3 +1,323 @@
...
+	"risuOpinions": "Les avis de RISU",
+	"risuTeam": "L'équipe de RISU",
+	"rowNumber": "Numéro de la ligne",
+	"rowNumberAsk": "Entrez la ligne du casier",
+	"rowNumbers": "Nombres de lignes",
```
Comment (hugolopespinto) [inline Configurateur/front/lib/l10n/app_fr.arb:280]: Nombre ?

File: Configurateur/front/lib/l10n/app_en.arb [hunk 5]
```diff
@@ -1,3 +1,324 @@
...
+	"registerResendEmail": "Resend confirmation email",
+	"registerRisu": "Register on the RISU website!",
+	"registerWith": "Register with:",
+	"registration": "Registration",
+	"remove": "Remove",
+	"removeLocker": "Remove a locker",
+	"returnTo": "Return",
+	"right": "Right",
+	"rightView": "Right view",
+	"risuCompany": "Risu Company",
```
Comment (hugolopespinto) [inline Configurateur/front/lib/l10n/app_en.arb:275]: Risu or RISU?

**Review by hugolopespinto — CHANGES_REQUESTED** [review]
just some fix in the translations, but otherwise good job!

**Review by H-chauvet — APPROVED** [review]

**Review by hugolopespinto — APPROVED** [review]

**Pull request merged**